diff -dur mu-conference-0.6.0.orig/jcr/src/main.c mu-conference-0.6.0/jcr/src/main.c
--- mu-conference-0.6.0.orig/jcr/src/main.c	2003-11-01 23:27:21.000000000 +0100
+++ mu-conference-0.6.0/jcr/src/main.c	2004-06-01 13:23:48.293786649 +0200
@@ -22,6 +22,11 @@
 
 #include "jcomp.h"
 
+#include <sys/types.h>
+#include <unistd.h>
+#include <grp.h>
+#include <pwd.h>
+
 int main(int argc, char *argv[]) {
   extern char *optarg;
   extern int optind, opterr, optopt;
@@ -33,6 +38,9 @@
   FILE *pid_stream;
   struct stat st;
   char *config_file = NULL;
+  char *user=NULL,*group=NULL;
+  struct passwd *pw;
+  struct group *gr;
   pool p;
   jcr = (jcr_instance)malloc(sizeof(_jcr_instance));
 
@@ -42,7 +50,7 @@
 
   fprintf(stderr, "%s -- %s\n%s\n\n", _JCOMP_NAME, _JCOMP_VERS, _JCOMP_COPY);
 
-  while ((c = getopt(argc, argv, "Bsd:c:")) != EOF)
+  while ((c = getopt(argc, argv, "Bsd:c:U:G:")) != EOF)
     switch (c) {
     case 'B':
       inBackground = 1;
@@ -57,6 +65,14 @@
       message_mask_set = 1;
       break;
 
+    case 'U':
+      user=optarg;
+      break;
+
+    case 'G':
+      group=optarg;
+      break;
+
     case 's':
       jcr->message_stderr = 1;
       message_stderr_set = 1;
@@ -117,6 +133,29 @@
   if (!message_stderr_set)
     jcr->message_stderr = (xmlnode_get_type(xmlnode_get_tag(jcr->config,"logstderr")) == NTYPE_TAG);
 
+  if (group) {
+	  gr=getgrnam(group);
+	  rc=setgid(gr->gr_gid);
+	  if (rc) {
+		  perror("setgid");
+		  return 1;
+	  }
+  }
+  if (user) {
+	  pw=getpwnam(user);
+	  if (!group) {
+		  rc=setgid(pw->pw_gid);
+		  if (rc) {
+			  perror("setgid");
+			  return 1;
+		  }
+	  }
+	  rc=setuid(pw->pw_uid);
+	  if (rc) {
+		  perror("setuid");
+		  return 1;
+	  }
+  }
 
   if (inBackground == 1) {
     if ((pid = fork()) == -1) {
@@ -141,8 +180,7 @@
   /* We now can initialize the resources */
   g_log_set_handler (NULL, G_LOG_LEVEL_MASK | G_LOG_FLAG_FATAL
                      | G_LOG_FLAG_RECURSION, jcr_log_handler, jcr);
-
-
+  
   log_warn(JDBG, "%s -- %s  starting.", _JCOMP_NAME, _JCOMP_VERS);
 
   config_file = xmlnode_get_data(xmlnode_get_tag(jcr->config,"pidfile"));

