diff -durN mu-conference-0.5.2.orig/src/Makefile mu-conference-0.5.2/src/Makefile
--- mu-conference-0.5.2.orig/src/Makefile	Mon Mar 10 08:03:07 2003
+++ mu-conference-0.5.2/src/Makefile	Mon Sep 15 12:42:53 2003
@@ -1,6 +1,6 @@
-include ../../platform-settings
+include /usr/lib/jabberd14/platform-settings
 
-CFLAGS:=$(CFLAGS) -I../../jabberd -I../include 
+CFLAGS:=$(CFLAGS) -I/usr/include/jabberd14 -I../include 
 
 # Debug/Experimental
 #CFLAGS:=$(CFLAGS) -pipe -Os -I../../jabberd -I../include 
