diff -ur mu-conference_0.7.orig/src/jabberd/Makefile mu-conference_0.7/src/jabberd/Makefile
--- mu-conference_0.7.orig/src/jabberd/Makefile	2007-06-08 22:19:48.000000000 +0200
+++ mu-conference_0.7/src/jabberd/Makefile	2008-05-11 22:51:15.000000000 +0200
@@ -1,6 +1,6 @@
 
 CC=gcc
-CFLAGS=-O2 -Wall -I. -I../../include `pkg-config --cflags glib-2.0` -D_REENTRANT -DLIBIDN
+CFLAGS=$(OFLAGS) -Wall -I. -I../../include `pkg-config --cflags glib-2.0` -D_REENTRANT -DLIBIDN
 LIBS=
 
 JCOMP_LIB_OBJECTS=expat.o \
diff -ur mu-conference_0.7.orig/src/jcomp/Makefile mu-conference_0.7/src/jcomp/Makefile
--- mu-conference_0.7.orig/src/jcomp/Makefile	2007-06-08 22:19:48.000000000 +0200
+++ mu-conference_0.7/src/jcomp/Makefile	2008-05-11 22:51:04.000000000 +0200
@@ -1,7 +1,7 @@
 
 #  $Id$
 CC=gcc
-CFLAGS=-O2 -Wall -I../../include -I. `pkg-config --cflags glib-2.0` -D_REENTRANT
+CFLAGS=$(OFLAGS) -Wall -I../../include -I. `pkg-config --cflags glib-2.0` -D_REENTRANT
 LIBS=
 
 JCOMP_OBJECTS=jcr_xdb.o \
diff -ur mu-conference_0.7.orig/src/Makefile mu-conference_0.7/src/Makefile
--- mu-conference_0.7.orig/src/Makefile	2007-06-08 22:19:48.000000000 +0200
+++ mu-conference_0.7/src/Makefile	2008-05-11 22:52:55.000000000 +0200
@@ -1,6 +1,6 @@
 
 CC:=gcc
-CFLAGS:=$(CFLAGS) -O2 -Wall -I../../lib -I../include `pkg-config --cflags glib-2.0` -D_JCOMP -D_REENTRANT -DLIBIDN
+CFLAGS:=$(CFLAGS) $(OFLAGS) -Wall -I../../lib -I../include `pkg-config --cflags glib-2.0` -D_JCOMP -D_REENTRANT -DLIBIDN
 #CFLAGS:=$(CFLAGS) -O2 -Wall -I../../lib -I../include `pkg-config --cflags glib-2.0` -D_JCOMP -D_REENTRANT -DLIBIDN -DHAVE_MYSQL
 LIBS:=$(LIBS) -ljcomp -lm `pkg-config --libs glib-2.0` `pkg-config --libs gthread-2.0` -lexpat -lidn `mysql_config --libs`
 LDFLAGS:=-L.
@@ -19,8 +19,8 @@
 	$(CC) $(CFLAGS) $(MCFLAGS) -o mu-conference $(conference_OBJECTS) $(LDFLAGS) $(LIBS)
 
 libjcomp.a:
-	cd jabberd ; make 
-	cd jcomp ; make lib
+	make -C jabberd
+	make -C jcomp lib
 
 static: $(conference_OBJECTS)
 
@@ -28,5 +28,5 @@
 
 clean:
 	rm -f $(conference_OBJECTS) mu-conference *~
-	cd jcomp ; make clean
-	cd jabberd ; make clean
+	make -C jabberd clean
+	make -C jcomp clean
